packages/compose-cli/src/bin.ts

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 import { runComposeCLI, spinnies } from './runComposeCLI.js';
-import 'ts-node/register';
 import 'dotenv/config';
+import 'tsx/cjs';
```

Comment:
That's interesting, why do we need to specifically load the CJS version  of tsx ?

Reply:
Because the bin itself is written in CJS.

https://github.com/ardatan/graphql-mesh/blob/afe0cc5ddfc7a1291dc878c61793b58850ae848b/packages/serve-cli/package.json#L15

Reply:
We can convert it to ESM if possible btw.

Reply:
If node doesn't have a problem with ESM executables, I am fine with changing this. **However,** I'd leave this for a separate PR not to overdo this one.